### PR TITLE
[ROCm] Fix for a build failure on the ROCm platform - 200420 

### PR DESCRIPTION
**Pull request by deven-amd**
Please see individual commit message for details.

/cc @chsigg @cheshire @nvining-work

### CODE REVIEW

**Comment by deven-amd** [comment]
@chsigg gentle ping

File: tensorflow/lite/kernels/pack_test.cc [hunk 1]
```diff
@@ -195,10 +195,10 @@ template <typename InputType>
 struct PackOpTestInt : public ::testing::Test {
   using TypeToTest = InputType;
   TensorType TENSOR_TYPE =
-      std::is_same<InputType, int16_t>::value
-          ? TensorType_INT16
-          : (std::is_same<InputType, uint8_t>::value ? TensorType_UINT8
-                                                     : TensorType_INT8);
+      (std::is_same<InputType, int16_t>::value
```
Comment (chsigg) [inline tensorflow/lite/kernels/pack_test.cc:198]: Please don't include unrelated changes like here and in unpack_test.cc.

**Review by chsigg — APPROVED** [review]

**Review by rthadur — APPROVED** [review]
